java/src/jmri/jmrit/symbolicprog/SplitEnumVariableValue.java

```diff
@@ -755,20 +755,42 @@ public void setLongValue(long value) {
 
         int lengthOfArray = this._valueArray.length;
 
+        boolean foundIt = false; // did we find entry? If not, have to add one
         for (int i = 0; i < lengthOfArray; i++) {
           if (this._valueArray[i] == value){
               log.trace("{} setLongValue setSelectedIndex to {}    <<<<<<<<<<<<<<<<", _name, i);
```

Comment:
You removed `<<<<<<<<<<<<<<<<` on line 893 in the method selectValue() but it's not removed here in the method setLongValue().

Reply:
Also, it's log.debug() on line 893 but log.trace() here.